lib/content-services/src/lib/search/components/search-filter-chips/search-widget-chip/search-widget-chip.component.spec.ts

```diff
@@ -65,4 +65,16 @@ describe('SearchWidgetChipComponent', () => {
       applyButton.triggerEventHandler('click', {});
       expect(queryBuilder.update).toHaveBeenCalled();
   });
+
+  it('should display arrow down icon',  () => {
+    const icon = fixture.debugElement.query(By.css('mat-chip mat-icon')).nativeElement.innerText;
+    expect(icon).toEqual('keyboard_arrow_down');
+});
```

Comment:
Probably this line and 79th are not aligned properly, at least it looks that here

Reply:
You're right, couple of places were misaligned, fixed now